src/test/java/com/ddd/moodof/acceptance/AcceptanceTest.java

```diff
@@ -71,13 +100,29 @@ void setUp() {
         }
     }
 
-    protected User signUp() {
-        return userRepository.save(new User(null, "test@test.com", "password", "nickname", "profileUrl", null, null, AuthProvider.google, "providerId"));
+    protected <T, U> U putWithLogin(T request, String uri, Class<U> response, Long userId) {
+        try {
+            String token = tokenProvider.createToken(userId);
+            String body = objectMapper.writeValueAsString(request);
+
+            MvcResult result = mockMvc.perform(MockMvcRequestBuilders.put(uri)
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .accept(MediaType.APPLICATION_JSON)
+                    .content(body)
+                    .header(AUTHORIZATION, BEARER + token))
+                    .andExpect(MockMvcResultMatchers.status().isCreated())
+                    .andExpect(MockMvcResultMatchers.header().string(HttpHeaders.LOCATION, Matchers.matchesRegex(uri + "/\\d*")))
+                    .andReturn();
```

Comment:
```suggestion
                    .andExpect(MockMvcResultMatchers.status().isOk())
                    .andReturn();
```

PUT 요청이니 Status가 Created보단 Ok가 되어야할 것 같아요

Reply:
이것도 리뷰코멘트 남긴사항과 유사해보여서 답변주시면 감사하겠습니다!